conf/application.conf

```diff
@@ -11,7 +11,7 @@ play.crypto.secret = "changeme"
 # ~~~~~
 play.i18n.langs = [ "en" ]
 
-ebean.default=["io.pathfinder.authentication.models.*"]
+ebean.default=["io.pathfinder.authentication.models.*", "io.pathfinder.pathfinderapplication.models.*"]
```

Comment:
What is the dfference between `io.pathfinder` and `io.pathfinder.pathfinderapplication`? In my opinion `io.pathfinder.pathfinderapplication` looks really ugly. Could we consolidate all models into `io.pathfinder.models` and all controllers into `io.pathfinder.controllers`?